### PR TITLE
feat(PWA): new-client PWA

### PR DESCRIPTION
**Pull request by josepot**
Initial work to have the basic caching and to unblock @juliezhudev .

### CODE REVIEW

File: src/new-client/src/sw.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+// You can also remove this file if you'd prefer not to use a
+// service worker, and the Workbox build step will be skipped.
+
+import { clientsClaim } from "workbox-core"
+import { ExpirationPlugin } from "workbox-expiration"
+import { precacheAndRoute, createHandlerBoundToURL } from "workbox-precaching"
+import { registerRoute } from "workbox-routing"
+import { StaleWhileRevalidate } from "workbox-strategies"
+
+clientsClaim()
```
Comment (iprado78) [inline src/new-client/src/sw.js:17]: What's tldr; on what this does?

Comment (juliezhudev) [reply src/new-client/src/sw.js:17]: Hey stranger :)

Comment (josepot) [reply src/new-client/src/sw.js:17]: Magic!

Comment (iprado78) [reply src/new-client/src/sw.js:17]: > Magic!
> 
> Just kidding, it's just that workbox kinda feels like magic 🙂, here you go: https://developers.google.com/web/tools/workbox/modules/workbox-core#clients_claim

🧙

**Review by juliezhudev — APPROVED** [review]

**Pull request merged**